src/router/Router.js

```diff
@@ -164,6 +167,18 @@ export default class Router extends DisposableBase {
             return this._observeEventsUsingConventions(modelId, object, methodPrefix);
         }
     }
+    addOnErrorHandler(handler) {
+        this._onErrorHandlers.push(handler);
```

Comment:
should  blow up if this isn't a function, i.e. simple `Guard.isFunction(handler)`. 

Looks like most other public methods have similar (except `runAction` which should).